multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/service/ObjectStoreServiceInfoCreator.java

```diff
@@ -6,57 +6,104 @@
 import java.util.Base64;
 import java.util.Map;
 
+import org.cloudfoundry.multiapps.controller.web.Constants;
 import org.cloudfoundry.multiapps.controller.web.Messages;
 import org.jclouds.domain.Credentials;
 import org.jclouds.googlecloud.GoogleCredentialsFromJson;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.google.common.base.Supplier;
```

Comment:
Sonarcloud reports for wrong used Supplier package, must probably `java.util.Supplier`

Reply:
this is the supplier that must be used, because google's jcloud provider works with it